grafana-plugin/src/plugin.json

```diff
@@ -192,6 +192,457 @@
       ]
     }
   ],
+  "roles": [
+    {
+      "role": {
+        "version": 1,
+        "name": "plugins.app:grafana-oncall-app:admin",
+        "displayName": "Admin",
+        "description": "Read/write access to everything in OnCall",
+        "group": "OnCall",
+        "permissions": [
+          { "action": "plugins.app:access", "scope": "plugins.app:id:grafana-oncall-app" },
```

Comment:
After discussing w/ Matvey, thinking of adding a  "OnCaller" role:
- view permissions to everything
- edit permissions to alert groups
- (maybe) edit permissions to schedules